lib/auth/password_test.go

```diff
@@ -403,7 +403,7 @@ func TestChangeUserAuthentication(t *testing.T) {
 			},
 		},
 		{
-			name: "with second factor webauthn",
```

Comment:
Should we keep the old test case and add a new one?

Reply:
this was a dupe test, but it was resolve in another PR